### ISSUE TITLE
Feature Request: Add code insertion point

### ISSUE THREAD

**Issue by florianbepunkt** [feathers-plus/cli#32]
I would like to extend a mongoose service like metioned here: https://docs.feathersjs.com/api/databases/common.html#classes-es6

However there is a code insertion point missing for the line:  ` app.use('/sequence', createService(options));`

```
class SequenceService extends MongooseService {
  // ommited for brevity
}

let moduleExports = function(app) {
  let Model = createModel(app);
  let paginate = app.get('paginate');
  // !code: func_init // !end

  let options = {
    Model,
    paginate
    // !code: options_more // !end
  };
  // !code: options_change // !end

  // Initialize our service with any options it requires
  // Could you please add a code insertion point for the follwong line?
  app.use('/sequence', new SequenceService(options));


  // Get our initialized service so that we can register hooks
  const service = app.service('sequence');

  service.hooks(hooks);
  // !code: func_return // !end
};

```

**Comment by eddyystop** [comment]
Certainly a worthwhile improvement. Thanks.

Resolved via PR https://github.com/feathers-plus/generator-feathers-plus/pull/243 .

The cli+ community has settled on posting issues to f+/generator-feathers-plus so please post any new issues there. Thanks.

**Comment by eddyystop** [comment]
Published in cli+ v0.8.6 which uses generator v0.8.6
